pom.xml

```diff
@@ -218,10 +218,11 @@
           <argLine>${jacoco.agent.arg}</argLine>
           <environmentVariables>
             <!-- Test service IPs outside the containers use the host's default IP address -->
+            <!-- The Auth Access Mode Service URI will have the ID to be checked appended onto the end -->
+            <AUTH_ACCESS_MODE_SERVICE>http://0.0.0.0:${test.hauth.port}/access</AUTH_ACCESS_MODE_SERVICE>
```

Comment:
Should these URLs (lines 222-226) be https instead of http?

Reply:
I think these are better as HTTP since these actually show up in the JSON response during manual testing. If they were HTTPS then we'd have to set up TLS in order for them to be usable during testing. I imagine in production we'd set these as HTTPS URLs anyway.